modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring the route admission policy
+
+By default, {microshift-short} allows routes in multiple namespaces to use the same hostname. To prevent routes in different namespaces from claiming the same hostname, you can configure the value.
+
+.Prerequisites
+
+* You created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. To prevent routes in different namespaces from claiming the same hostname, update the `namespaceOwnership` field value to `Strict` in the {microshift-short} `config.yaml` file as shown in the following example:
++
+.Example `config.yaml` route admission policy
+[source,yaml]
+----
+# ...
+ingress:
+  routeAdmissionPolicy:
+    namespaceOwnership: Strict <1>
+# ...
+----
+<1> Valid values are `Strict` and `InterNamespaceAllowed`. Specifying `Strict` prevents routes in different namespaces from claiming the same host. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
++
+[TIP]
+====
+If you complete all the configurations that you need to make in the {microshift-short} `config.yaml` file at the same time, you can minimize system restarts.
+====
+
+. Restart the {microshift-short} service by running the following command:
```

Comment:
Can we possibly provide a short reason why restart the service? 
```suggestion
. To apply the configuration, restart the {microshift-short} service by running the following command:
```

Reply:
I am retaining this Tip as is to maintain consistency with this https://github.com/openshift/openshift-docs/pull/76100 PR.

Reply:
Same comment https://github.com/openshift/openshift-docs/pull/76317#discussion_r1625204882

Reply:
made this change. done.